<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionLoad_Input_Mesh"/>
    <addaction name="actionLoad_Mesh_Annotation"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCRF"/>
    <addaction name="actionCRF3D"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Input Image</string>
   </property>
  </action>
  <action name="actionLoad_Annotation">
   <property name="text">
    <string>Load Annotation Image</string>
   </property>
  </action>
  <action name="actionCRF">
   <property name="text">
    <string>CRF2D</string>
   </property>
  </action>
  <action name="actionCRF3D">
   <property name="text">
    <string>CRF3D</string>
   </property>
  </action>
  <action name="actionLoad_Input_Mesh">
   <property name="text">
    <string>Load Input Mesh</string>
   </property>
  </action>
  <action name="actionLoad_Mesh_Annotation">
   <property name="text">
    <string>Load Annotation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
